src/App.tsx

```diff
@@ -1,61 +1,121 @@
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export type Todo = {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+  user: {
+    id: number;
+    name: string;
+    username: string;
+    email: string;
+  } | null;
```

Comment:
```suggestion
  user: User | null;
```
better to move all types to separate file